scripts/db_migrator.py

```diff
@@ -509,6 +509,18 @@ def migrate_config_db_port_table_for_auto_neg(self):
                 elif value['autoneg'] == '0':
                     self.configDB.set(self.configDB.CONFIG_DB, '{}|{}'.format(table_name, key), 'autoneg', 'off')
 
+    def migrate_config_db_port_table_for_dhcp_rate_limit(self):
+        port_table_name = 'PORT'
+        port_table = self.configDB.get_table(port_table_name)
+
+        for p_key, p_value in port_table.items():
+            if 'dhcp_rate_limit' in p_value:
+                self.configDB.set(self.configDB.CONFIG_DB, '{}|{}'.format(port_table_name, p_key),
+                                  'dhcp_rate_limit', p_value['dhcp_rate_limit'])
+            else:
+                self.configDB.set(self.configDB.CONFIG_DB, '{}|{}'.format(port_table_name, p_key),
```

Comment:
Currently copp rate for Mgmt device is 300, and other are 100 https://github.com/sonic-net/sonic-buildimage/blob/a7ab698f1c7218b4ddc4db63c42918a8c3eb9eb4/files/image_config/copp/copp_cfg.j2#L37-L43. Can we align with it?

Reply:
Hi @yaqiangz , as per our HLD document, we plan to depreciate the default COPP DHCP rate limit and replace it with our new rate-limiting mechanism since TC holds and advantage over COPP in the event of a DoS attack (it can rate limit at an interface level). We kept the default value 300 packets/sec to ensure backward compatibility with the depreciated COPP limit. We have added a sonic-buildimage PR to do this which is linked to our main HLD PR.